website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,9 +9,11 @@ description: >-
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-08-03
+* Add `provisional` to `configuration-versions` endpoint
```

Comment:
```suggestion
* Add `provisional` to `configuration-versions` endpoint.
```

Reply:
so it matches the other updates in the changelog that end up with a period at the end.